ansible/roles/schulcloud-server-core/tasks/main.yml

```diff
@@ -240,6 +240,24 @@
     tags:
       - configmap
 
+  - name: Data deletion trigger failed CronJob
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: data-deletion-trigger-failed-cronjob.yml.j2
```

Comment:
I'm not a huge fan of naming a cronjob "cronjob" in the file name, but thats more of a general thought regarding that topic